app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -504,10 +504,6 @@ def self.default_blacklisted_event_names
       )
     end
 
-    def build_availability_metric_id(feed_id, resource_id, metric_id)
-      "AI~R~[#{feed_id}/#{resource_id}]~AT~#{metric_id}"
```

Comment:
Good removal.
This built id logic could change, so fetching directly from inventory via type id is a good change.